src/Installer.php

```diff
@@ -72,12 +97,27 @@ private function installDocumentType(array $typeDefinition): void
 
     private function installThumbnailConfiguration(): void
     {
-        if (Thumbnail\Config::exists('pimcore-presentation-bundle-background-image')) {
+        if (ThumbnailConfig::exists(self::THUMBNAIL_CONFIG_BACKGROUND_IMAGE)) {
             return;
         }
 
-        $thumbnailConfig = new Thumbnail\Config();
-        $thumbnailConfig->setName('pimcore-presentation-bundle-background-image');
+        $thumbnailConfig = new ThumbnailConfig();
+        $thumbnailConfig->setName(self::THUMBNAIL_CONFIG_BACKGROUND_IMAGE);
         $thumbnailConfig->save();
     }
+
+    private function uninstallThumbnailConfiguration(): void
+    {
+        if ($thumbnailConfig = ThumbnailConfig::getByName(self::THUMBNAIL_CONFIG_BACKGROUND_IMAGE)) {
+            $thumbnailConfig->delete();
+        }
+    }
+
+    private function getTypeDefinitionsFile(): string
+    {
+        return __DIR__ . \DIRECTORY_SEPARATOR .
+            '..' . \DIRECTORY_SEPARATOR .
+            'config' . \DIRECTORY_SEPARATOR .
+            'document-types.php';
+    }
```

Comment:
Although I would actually do it like this:
```suggestion
    private function getDocumentTypeDefinitions(): array
    {
        return require dirname(__DIR__) . \DIRECTORY_SEPARATOR .
            'config' . \DIRECTORY_SEPARATOR .
            'document-types.php';
    }
```

Then you don't need the `include` when using it.